src/main/java/edu/kis/powp/jobs2d/drivers/DriverFeaturesManager.java

```diff
@@ -0,0 +1,37 @@
+package edu.kis.powp.jobs2d.drivers;
+
+import edu.kis.legacy.drawer.shape.line.BasicLine;
+import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.drivers.adapter.LineDriverAdapter;
+import edu.kis.powp.observer.Publisher;
+
+public class DriverFeaturesManager extends DriverManager {
+    private DriversComposite currentDriver = new DriversComposite();
+    private final Publisher changePublisher = new Publisher();
+    private final static Job2dDriver PREVIEW_DRIVER = new LineDriverAdapter(null, new BasicLine(),"preview");
+
+    public synchronized void setCurrentDriver(Job2dDriver driver) {
+        if (currentDriver.hasDriver(driver)) {
+            currentDriver.removeDriver(driver);
+        }
```

Comment:
Now selecting the same driver (e.g., "Basic line driver") twice disables the driver. This approach does not make sense!